environments/autostart.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Autostart"
+description: Learn how to start your environments automatically at a predetermined time.
+---
+
+Autostart allows you to set environments to turn on automatically at a
+predetermined time.
+
+## Background
+
+Coder automatically turns off unused environments after 8 hours to help you save
```

Comment:
Technically, this is scoped per-organization - we just happen to use `8h` internally.

![Screenshot from 2021-03-16 12-16-43](https://user-images.githubusercontent.com/11184711/111351759-83677300-8651-11eb-8f2f-c46abe0b870c.png)

Reply:
We changed it the auto-off threshold to default to 8 now as well.

Reply:
```suggestion
Coder automatically turns off unused environments after 8 hours by default to help you save
```

Reply:
^ works for me!